.github/.labeler.yml

```diff
@@ -0,0 +1,59 @@
+# Add 'Project: ngrx.io' label to any ngrx.io file changes
+'Project: ngrx.io':
+  - projets/ngrx.io/*
```

Comment:
Can you fix the typo `projets` to `projects` please?
Here, and also the other occurrences.